From 936dbe1582e6b7f9b84392b90ce148c1064b2bda Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 19 May 2025 06:22:56 +0300
Subject: [PATCH] [SW-0] gaudi3: hl_npu: remove unused code

This patch removes unused hw_isr_work handler
in gaudi3.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I8abe32a03d504986b00279e8a7bdf652eb6a712a
---

diff --git a/drivers/misc/hl_npu/gaudi3/hl.c b/drivers/misc/hl_npu/gaudi3/hl.c
index 907b4ee..8bb6997 100644
--- a/drivers/misc/hl_npu/gaudi3/hl.c
+++ b/drivers/misc/hl_npu/gaudi3/hl.c
@@ -109,8 +109,6 @@
 
 static struct gaudi3_eq_events eq_events;
 
-static struct k_work hw_isr_work;
-
 static struct gaudi3_priv_data gaudi3_priv;
 
 static struct hl_drv_data gaudi3_drv_data = {
@@ -425,11 +423,6 @@
 {
 }
 
-static void hl_work_hw_isr_handler(struct k_work *work)
-{
-	/* TODO */
-}
-
 static int hl_gaudi3_irq_config(const struct device *dev)
 {
 	/* Register for Interrupts from Host using the SW INTR index */
@@ -605,7 +598,6 @@
 	 */
 	memset(&eq_events, 0, sizeof(struct gaudi3_eq_events));
 	eq_events.size = GAUDI3_HW_EVENTS_HISTORY_BUF_SZ;
-	k_work_init(&hw_isr_work, hl_work_hw_isr_handler);
 	k_work_init(&reset_work, hl_reset_handler);
 
 	/* set the values of power max */
